src/sysusers/sysusers.c

```diff
@@ -64,13 +64,19 @@ typedef struct Item {
         uid_t uid;
 
         bool gid_set:1;
-        bool gid_must_exist:1;
+        bool gid_must_exist_and_can_be_reused:1;
```

Comment:
What a long name... The length of the name is not problem, but I think the name is slightly gapped from the logic what you added in the previous PR and this. Are there any suggestion?

Reply:
Indeed, how about `id_set_strict` as a strawman?  I pushed the PR with this. Happy to think about more alternatives here if you prefer a different one. There is this old joke about the three hardest problems in computer science: 1. naming things 2. off-by-one errors.